Documentation/Input/Pointers.md

```diff
@@ -106,8 +106,9 @@ The *[SpherePointer](xref:Microsoft.MixedReality.Toolkit.Input.SpherePointer)* u
 Useful Sphere Pointer properties:
 
 - *Sphere Cast Radius*: The radius for the sphere used to query for grabbable objects.
-- *Near Object Margin*: The distance to query for detecting of an object is near the pointer.
+- *Near Object Margin*: The distance on top of the Sphere Cast Radius to query for detecting if an object is near the pointer. Total Near Object detection radius is Sphere Cast Radius + Near Object Margin
```

Comment:
Do we need to update any pictures here also to communicate that the default is a cone, not a sphere?

Reply:
Will do